client/blocks/upe/upe-deferred-intent-creation/payment-processor.js

```diff
@@ -269,7 +291,7 @@ const PaymentProcessor = ( {
 			/>
 			<PaymentElement
 				options={ getStripeElementOptions() }
-				onChange={ updatePaymentElementCompletionStatus }
+				onChange={ onChange }
```

Comment:
Same as above...  `onSelectedPaymentMethodChange` or similar
```suggestion
				onChange={ onSelectedPaymentMethodChange }
```